Kong Mesh license (#3680)
Kong/docs.konghq.com: app/mesh/1.5.x/license.md

​
```
Number of DPPs = Number of Pods + Number of VMs.
```
​
With a valid issued license, a data plane proxy will always be able to join the service mesh even if we go above the allowed limit in order to prevent service disruptions (a warning in the GUI and in the logs will identify this situation to the end user). On the other end, with the prebundled license, whenever we go over the maximum allowed number of DPPs the system will automatically refuse their connection.
​
## License API
​
The license can be inspected via the GUI or an API at the `/license` endpoint on the control plane. For example:
​
```
$ curl <IP of the control plane>:5681/license
{
 "allowedDataplaneProxies": 20,
 "activeDataplaneProxies": 2,
 "expirationDate": "2032-11-09T00:00:00Z",
 "demo": false
}
```
​
## Configuring the license
​
A valid license file can be passed to Kong Mesh in a variety of different ways.
​
### Kubernetes (kumactl)
​
When installing the Kong Mesh control plane with `kumactl install control-plane`, please provide a `--license-path` argument with a full path to a valid license file, for example:
​
```sh
$ kumactl install control-plane --license-path=/path/to/license.json
```
​

> ### Kubernetes (HELM)

Reviewer: ```suggestion
### Kubernetes (Helm)
```

​

> To install a valid license via HELM, please follow these instructions:

Reviewer: ```suggestion
To install a valid license via Helm:
```

​
1) Create a secret named `kong-mesh-license` in the same namespace where Kong Mesh is being installed. For example:
​
```sh
$ kubectl create namespace kong-mesh-system
$ kubectl create secret generic kong-mesh-license -n kong-mesh-system --from-file=/path/to/license.json
```
​

> Where `kong-mesh-system` is the namespace where Kong Mesh control plane is installed and `/path/to/license.json` is the path to a valid license file. Note that the file name should be `license.json` unless otherwise specified in `values.yaml`.

Reviewer: ```suggestion
Where:
* `kong-mesh-system` is the namespace where Kong Mesh control plane is installed
* `/path/to/license.json` is the path to a valid license file. The filename should be `license.json` unless otherwise specified in `values.yaml`.
```

​
2) Modify the `values.yaml` file to point to the secret. For example:
​
```yaml
kuma:
  controlPlane:
    secrets:
      - Env: "KMESH_LICENSE_INLINE"
        Secret: "kong-mesh-license"
        Key: "license.json"
```
​
### Universal
​

> In Universal, we can configure a valid license by using the following environment variables:

Reviewer: ```suggestion
In Universal mode, configure a valid license by using the following environment variables:
```
Is it Universal mode, or is the term something else? It needs some sort of noun here.

​
* `KMESH_LICENSE_PATH` - valued with the path to a valid license file.
* `KMESH_LICENSE_INLINE` - valued with the actual contents of the license file.
​
## Multi-zone
​

> As described earlier, in a multi-zone deployment of Kong Mesh only the global control plane should be configured with a valid license. The global control plane will automatically synchronize the license to any remote control plane that is part of the cluster.

Reviewer: ```suggestion
In a multi-zone deployment of Kong Mesh, only the global control plane should be configured with a valid license. The global control plane automatically synchronizes the license to any remote control plane that is part of the cluster.
```
Try not to refer to "earlier" or "later" (or above/below) in a doc, as people will often not read content in order, and the order could always change.
